modules/user/client-react/components/LoginView.native.jsx

```diff
@@ -16,19 +16,30 @@ class LoginView extends React.PureComponent {
   }
 
   componentWillUnmount() {
-    Linking.removeListener('url');
+    Linking.removeEventListener('url', this.handleOpenURL);
   }
 
   handleOpenURL = async ({ url }) => {
+    if (!url.match(/data=([^#]+)/)) return;
```

Comment:
Use constant for regexp to avoid duplications.

Reply:
done